### DISCUSSION TITLE
Got a Question? Post it here

### DISCUSSION THREAD

**Discussion by rparakhiya** [tark-ignite/codegenie-2023#2] [Q&A]
Use this thread to ask questions. Someone from Tark will post clarifications.

**Comment by uttamsutariya** [comment]
I think in StudentScreener question there is no need of variable `totalMarks` as it's not used in any calculation of percentile or anywhere else, because pr is calculated based on students rank. I may be wrong but according to me that's need not to be there.

**Reply by vimal-parakhiya** [reply]
That's a good catch, we do not need `totalMarks` to solve `StudentScreener` problem!

**Comment by khunt-miral** [comment]
In practice-problem-statements.pdf, In StudentScreener problem requiredPercentile is given in Integer but in skeletons it is given in float so it would be better if it was same at both places.

**Reply by vimal-parakhiya** [reply]
Fair point, we will keep this in mind, going forward!

**Comment by Saurav-Sutaria** [comment]
In compiler demonstration video there is only compile & test button, there is no submit code button, so can we assume that by clicking on compile & test button we have made the submission for evaluation for a given problem?

**Reply by rparakhiya** [reply]
The video is intended to demonstrate the workflow of writing, compiling and testing the code. In the actual event, you'd see a separate button to submit your solution once you have written and tested it.

**Reply by Saurav-Sutaria** [reply]
Ok, got it.

**Comment by uttamsutariya** [comment]
Can you share all the test cases of both problems ?

**Reply by vimal-parakhiya** [reply]
It will be difficult to share all the test cases for confidentiality reasons! However, during next round, we can have 1:1 discussion and help you in identifying the mistakes in your solution (towards end of the session, if time permits).

By the way, congratulations for being selected for the next round.

**Comment by Saurav-Sutaria** [comment]
Sir, is the result declared?

**Reply by rparakhiya** [reply]
@Saurav-Sutaria Yes, results are in! You can find list of winners and selected for the final rounds at https://tarktech.com/codegenie-2023/

Congratulations to you for being one of top 20! 🎊

**Reply by Saurav-Sutaria** [reply]
Thank you sir
